frontend/app/helpers/spree/frontend_helper.rb

```diff
@@ -255,6 +255,14 @@ def static_filters
       @static_filters ||= Spree::Frontend::Config[:products_filters]
     end
 
+    def additional_filters_partials
```

Comment:
@AleksandraKaminska nice ide - let's leave `additional_filters_partials` config only, we don't need separate mobile because we can...